### PR TITLE
Makes the morgue alt-click message clearer

### PR DESCRIPTION
**Pull request by nicbn**

### CODE REVIEW

**Comment by ExcessiveUseOfCobblestone** [comment]
I want the shed to be red1!!!

**Review by ShizCalev — APPROVED** [review]

**Comment by DaedalusGame** [comment]
>using up 1 gbp to reorder some words

**Comment by PKPenguin321** [comment]
This should be `Grammar/Formatting`, not Tweak

File: code/game/objects/structures/morgue.dm [hunk 1]
```diff
@@ -153,7 +153,7 @@ GLOBAL_LIST_EMPTY(bodycontainers) //Let them act as spawnpoints for revenants an
 
 /obj/structure/bodycontainer/morgue/examine(mob/user)
 	..()
-	to_chat(user, "<span class='notice'>Alt-click [src] to [beeper ? "disable" : "enable"] the alarm.</span>")
+	to_chat(user, "<span class='notice'>The speaker is [beeper ? "enabled" : "disabled"]. Alt-click to toggle it.</span>")
```
Comment (MoreRobustThanYou) [inline code/game/objects/structures/morgue.dm]: change ```notice``` to ```notice bold```

Comment (ExcessiveUseOfCobblestone) [reply code/game/objects/structures/morgue.dm]: why don't you just bold the enabled/disabled if you're going to do that.

**Comment by nicbn** [comment]
> thinking i care about gbp

File: code/game/objects/structures/morgue.dm (hunk 1 above)
Comment (nicbn) [reply code/game/objects/structures/morgue.dm]: It's not important enough to deserve it

**Pull request merged**